regression_test.py

```diff
@@ -147,6 +147,7 @@ def launch_tests():
             Test("CPPCLI", "CLRTest.sharpmake.cs"),
             Test("CSharpHelloWorld", "HelloWorld.sharpmake.cs"),
             Test("HelloWorld", "HelloWorld.sharpmake.cs"),
+            Test("JumboBuild", "JumboBuild.sharpmake.cs"),
             Test("HelloLinux", "HelloLinux.Main.sharpmake.cs"),
```

Comment:
you need to modify update UpdateSamplesOutput.bat as well to be able update the reference files.